src/components/presentational/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,51 @@
+.mdhui-progress-bar {
+	display: block;
+	width: 100%;
+	position: relative;
+	--icon_size: 24px;
+	--icon_border_width: 2px;
+}
+
+.mdhui-progress-bar .mdhui-progress-bar-default-margin {
```

Comment:
Looks like this isn't working because you have a space between these and this class is applied to the progress bar itself.

Also, you will need to adjust the width like we do for buttons so it's ```width:calc(100% - 32px)```, otherwise it overflows